client/components/DetailViewContainer.js

```diff
@@ -102,28 +103,93 @@ CarouselItem.propTypes = {
   }).isRequired
 };
 
-const Factoid = ({ title, message, headerMessage, headerTitle, link }) => (
-  <li
-    className="c_home__factoid"
-  >
-    <div className="c_home__factoid__goldHeader">
-      <div className="c_home__factoid__title">
-        {headerTitle}
+const REASONS = [
+  <FormattedMessage
+    id="components.DetailViewContainer.top_reasons.item.1"
+    defaultMessage="Determines the number of seats in the House of Representatives">
+  </FormattedMessage>,
+  <FormattedMessage
+    id="components.DetailViewContainer.top_reasons.item.2"
+    defaultMessage="Redraw district boundaries">
+  </FormattedMessage>,
+  <FormattedMessage
+    id="components.DetailViewContainer.top_reasons.item.3"
+    defaultMessage="Allocates funds to the state and localities">
+  </FormattedMessage>,
+  <FormattedMessage
+    id="components.DetailViewContainer.top_reasons.item.4"
+    defaultMessage="Infrastructure planning">
+  </FormattedMessage>,
+  <FormattedMessage
+    id="Compnents.DetailViewContainer.top_reasons.item.4"
+    defaultMessage="Emergency response planning">
+  </FormattedMessage>,
+]
+
+const TopReasons = () => {
+  let screen_width = window.innerWidth
+  let size
+  if (screen_width < 1200 ) {
+    size = (screen_width < 800) ? 'small' : 'medium'
+  } else {
+    size = 'large'
+  }
+  let top_reasons_class = "c_home__top-reasons"
+
+  return (
+    <ol
+      className={classnames(
+        top_reasons_class,
+        `${top_reasons_class}__${size}`,
```

Comment:
Good application of BEM! One minor thing, the "official" [BEM specs](http://getbem.com/naming/) use `--` as a separator for modifiers like ${size}. You could change this to `${top_reasons_class}--${size}`.